src/App/utils/RemoveVscroll/index.js

```diff
@@ -0,0 +1,5 @@
+const RemoveVscroll = () => {
+    document.body.classList.add("no-vscroll");
+};
+
+export default RemoveVscroll;
```

Comment:
heh I guess this can replace the code used at the sheet component and also added to the dialog component to fix [this issue](https://github.com/SwedbankPay/design.swedbankpay.com/issues/337)